src/sugar3/activity/bundlebuilder.py

```diff
@@ -144,6 +144,11 @@ def build_locale(self):
             cat = gettext.GNUTranslations(open(mo_file, 'r'))
             translated_name = cat.gettext(self.config.activity_name)
             translated_summary = cat.gettext(self.config.summary)
+            if translated_summary.find('\n') > -1:
+                translated_summary = translated_summary.replace('\n', '')
+                logging.warn(
+                    'Translation of summary on file %s have \\n chars.'
+                    'Should be removed' % file_name)
```

Comment:
Awesome patch!    Can we just put a space here so there is a space after the full stop in the log message?   Just a very small thing sorry :crying_cat_face: